tests/test_disassembly.py

```diff
@@ -1,21 +1,22 @@
 # pylint:disable=missing-class-docstring,no-self-use
 from unittest import TestCase, main
 
+import archinfo
 import angr
 from angr.analyses import Disassembly
 from angr.analyses.disassembly import MemoryOperand, Instruction
 
 
 class TestDisassembly(TestCase):
-    def test_arm64_disect_instruction(self):
+    def test_arm_dissect_instructions(self):
         proj = angr.load_shellcode(
             b"\x00\xe4\x00\x6f"
             b"\x43\x3c\x0b\x0e"
             b"\x54\x9a\xb7\x72"
             b"\xfc\x6f\xba\xa9"
             b"\x88\x03\x98\x1a"
             b"\x00\x60\x01\x4e",
-            "AARCH64",
+            archinfo.ArchAArch64(),
```

Comment:
IMO this change is not necessary. angr (in fact, archinfo) should be able to convert "AArch64" to `archinfo.ArchAArch64`.